test/test_rospkg_environment.py

```diff
@@ -54,13 +54,13 @@ def test_get_ros_root():
     
 def test_get_ros_package_path():
     from rospkg import get_ros_package_path
-    assert None == get_ros_package_path(env={})
-    env = {'ROS_PACKAGE_PATH': ':'}
-    assert ':' == get_ros_package_path(env=env)
+    assert ':/usr/share' == get_ros_package_path(env={})
+    env = {'ROS_PACKAGE_PATH': ''}
```

Comment:
I don't think removing the `:` in the example environment is the right way to make this test pass. In the current patch, if `ROS_PACKAGE_PATH` is set to `:` then `get_ros_package_path()` would return `::/usr/share`. The test should reflect that or the code should ensure that no redundant `:` are left in the result.